xarray/plot/utils.py

```diff
@@ -675,47 +677,48 @@ def _valid_numpy_subdtype(x, numpy_types):
     return any(np.issubdtype(x.dtype, t) for t in numpy_types)
 
 
-def _ensure_plottable(*args):
+def _ensure_plottable(*args) -> None:
     """
     Raise exception if there is anything in args that can't be plotted on an
     axis by matplotlib.
     """
-    numpy_types = [
+    numpy_types: tuple[type[object], ...] = (
         np.floating,
         np.integer,
         np.timedelta64,
         np.datetime64,
         np.bool_,
         np.str_,
-    ]
-    other_types = [datetime]
+    )
+    other_types: tuple[type[object], ...] = (datetime,)
```

Comment:
```suggestion
    other_types: tuple[type[object], ...] = (datetime,)
    cftime_datetime_types: tuple[Any, ...] = ()
```
Maybe `Any` can be replaced with `cftime.datetime`?
Can also remove the else below with this.

Reply:
Why do you want to use Any? I wanted to highlight that this is a tuple of classes, not instances.